### ISSUE TITLE
Add Imprint form

### ISSUE THREAD

**Issue by timoludwig** [Integreat/cms-django#358]
Add a form to edit the imprint/disclaimer.
This involves the form-model, form views and form templates.
This should be similar to the page form, please do not simply copy + paste, but search ways to generalize and re-use existing code.

Depends on #357.

**Comment by timoludwig** [comment]
Possible duplicate of #39

**Comment by timoludwig** [comment]
Duplicate of #39
